index.html

```diff
@@ -41,7 +46,169 @@ <h2 class="enphasis">I'm a software developer</h2>
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
+        <section class="card">
+            <section class="screenshot">
+                <img src="/Images/placeholder.png" alt="screenshot">
+            </section>
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
+        <section class="card">
+            <section class="screenshot">
+                <img src="/Images/placeholder.png" alt="screenshot">
+            </section>
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>Ruby on rails</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
+        <section class="card">
+            <section class="screenshot">
+                <img src="/Images/placeholder.png" alt="screenshot">
+            </section>
+            
+            <section class="info">
+                <h3 class="text">
+                    Multi-Post Stories Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Ruby on rails</li>
+                </ul>
+                <a class="button" href="">See project</a>
+            </section>
+        </section>
+
     </section>
+
+
+    <section class="about">
+        <section class="aboutme">
+            <h1>About me</h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+             <a class="button" href="">Get My Resume</a>
+        </section>
+
+        <section class="card">
+            <img src="/Images/languages-icon.svg" alt="languages">
+            <h3>Languages</h3>
+            <ul class="tags">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+        </section>
```

Comment:
# Optional but Important
- [ ] [OPTIONAL] According to project requirement your card `section` should be `ul`  element and `framework`, `skills` and `language` should be wrapped in `li` element, kindly fix that

Reply:
Thanks for that! I completely forgot about it.